This allows lzma-sys to build in a cross-compiling environment.
--- rustc-1.36.0-src/vendor/lzma-sys/build.rs
+++ rustc-1.36.0-src/vendor/lzma-sys/build.rs
@@ -101,6 +101,7 @@ fn main() {
         }
         cmd.env("CC", compiler.path())
            .env("CFLAGS", cflags)
+           .env("LDFLAGS", "")
            .current_dir(&dst.join("build"))
            .arg(sanitize_sh(&src.join("configure")));
         cmd.arg(format!("--prefix={}", sanitize_sh(&dst)));
